spec/solidus_quiet_logistics/outbound/document/shipment_order_spec.rb

```diff
@@ -56,8 +56,8 @@
           <OrderHeader OrderNumber="#{shipment.number}" OrderType="SO" CustomerPO="#{shipment.number}" VIPCustomer="false" StoreDelivery="false" Gift="#{gift_message.present?}" OrderPriority="#{order_priority}" OrderDate="#{order_date}">
             <Comments>#{gift_message}</Comments>
             <ShipMode Carrier="#{shipping_method.carrier}" ServiceLevel="#{shipping_method.service_level}"/>
-            <ShipTo Contact="#{ship_address.full_name}" Address1="#{ship_address.address1}" Address2="#{ship_address.address2}" City="#{ship_address.city}" State="#{ship_address.state.name}" PostalCode="#{ship_address.zipcode}" Country="#{ship_address.country.iso}" Phone="#{ship_address.phone}" Email="#{order.email}"/>
-            <BillTo Contact="#{bill_address.full_name}" Address1="#{bill_address.address1}" Address2="#{bill_address.address2}" City="#{bill_address.city}" State="#{bill_address.state.name}" PostalCode="#{bill_address.zipcode}" Country="#{bill_address.country.iso}" Phone="#{bill_address.phone}" Email="#{order.email}"/>
+            <ShipTo Contact="#{ship_address.respond_to?(:name) ? ship_address.name : ship_address.full_name}" Address1="#{ship_address.address1}" Address2="#{ship_address.address2}" City="#{ship_address.city}" State="#{ship_address.state.name}" PostalCode="#{ship_address.zipcode}" Country="#{ship_address.country.iso}" Phone="#{ship_address.phone}" Email="#{order.email}"/>
+            <BillTo Contact="#{bill_address.respond_to?(:name) ? bill_address.name : bill_address.full_name}" Address1="#{bill_address.address1}" Address2="#{bill_address.address2}" City="#{bill_address.city}" State="#{bill_address.state.name}" PostalCode="#{bill_address.zipcode}" Country="#{bill_address.country.iso}" Phone="#{bill_address.phone}" Email="#{order.email}"/>
```

Comment:
I think we need to check with `SolidusSupport.combined_first_and_last_name_in_address?` here

Reply:
It would be more explicit and readable. I can certainly make the change